### PR TITLE
Update background default colors

### PR DESCRIPTION
**Pull request by marciopmoreira6**

### CODE REVIEW

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
LGTM :+1:

File: src/generated/design-tokens-dark.css [hunk 1]
```diff
@@ -104,9 +104,4 @@ html[data-echoes-theme='dark'], [data-echoes-theme='dark'] {
   --echoes-box-shadow-medium: inset 0 2px 15px -2px rgba(8,9,12,0.06), 0 4px 15px -2px rgba(8,9,12,0.5);
   --echoes-box-shadow-large: 0 4px 6px -2px rgba(8,9,12,0.05), 0 12px 20px -4px rgba(8,9,12,0.5);
   --echoes-box-shadow-x-large: 0 8px 8px -4px rgba(8,9,12,0.04), 0 20px 30px -4px rgba(8,9,12,0.5);
-  --echoes-sizes-overlays-max-width-default: 25rem;
-  --echoes-sizes-inputs-small: 150px;
-  --echoes-sizes-inputs-medium: 250px;
-  --echoes-sizes-inputs-large: 350px;
-  --echoes-sizes-inputs-full: 100%;
```
Comment (jeremy-davis-sonarsource) [inline src/generated/design-tokens-dark.css:111]: Don't we need these?

File: design-tokens/tokens/$themes.json [hunk 2]
```diff
@@ -6,89 +6,90 @@
     "selectedTokenSets": {
       "layer1/colors": "source",
       "layer2/light": "enabled",
-      "layer3/base": "enabled",
-      "layer3/light": "enabled"
+      "layer3/light": "enabled",
+      "layer1/base": "source"
     },
```
Comment (jeremy-davis-sonarsource) [inline design-tokens/tokens/$themes.json:11]: I think this is the issue: we aren't using the layer3 base?

**Review by jeremy-davis-sonarsource — CHANGES_REQUESTED** [review]

File: design-tokens/tokens/$themes.json (hunk 2 above)
Comment (david-cho-lerat-sonarsource) [reply design-tokens/tokens/$themes.json:11]: oh, maybe this PR should be rebased on our common branch?

Comment (marciopmoreira6) [reply design-tokens/tokens/$themes.json:11]: I enabled layer 3 - base on the base theme. Can you check if this is fixed now @jeremy-davis-sonarsource ? Thanks

**Review by jeremy-davis-sonarsource — APPROVED** [review]

**Pull request merged**